<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SlaveWidget</class>
 <widget class="QWidget" name="SlaveWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="connectionGroupBox">
     <property name="title">
      <string>Master Connection</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="serverLabel">
        <property name="text">
         <string>Server:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="serverAddressEdit">
        <property name="text">
         <string>localhost</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="portLabel">
        <property name="text">
         <string>Port:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="portSpinBox">
        <property name="minimum">
         <number>1024</number>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>8080</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="connectButton">
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="disconnectButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Disconnect</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="progressGroupBox">
     <property name="title">
      <string>Calculation Progress</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="logGroupBox">
     <property name="title">
      <string>Log</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QTextEdit" name="logTextEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="statusLabel">
     <property name="text">
      <string>Not connected</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
